src/biblebot/constants/__init__.py

```
@@ -0,0 +1,18 @@
+"""
+Constants package for BibleBot.
+
+This package follows the mmrelay pattern of organizing constants into
+separate files by category. This __init__.py re-exports a subset of
+the most commonly used constants for convenience, allowing for imports
+like `from biblebot.constants import APP_NAME`.
+"""
+
+# ruff: noqa: F401, F403
```

Ruff (0.12.2)

Unused `noqa` directive (non-enabled: `F401`)

Remove unused `noqa` directive

(RUF100)